draft-ietf-rats-architecture.md

```diff
@@ -1317,3 +1317,160 @@ use beyond the period for which it deems the Attestation Result to remain
+* Adds their own SW to the device over the top of the SW the chip vendor supplied
+* Operates a cloud-based service with which the device interacts
+* An Attestation Evidence is included whenever the IoT device communicates with the cloud service
+* The cloud service decodes the Attestation Evidence to get the key ID and computes the HMAC for it
+* The close service sends the key ID and HMAC to the chip vendors verification service
+
+This maps as follows in the RATS architecture:
+
+Endorser:
+
+: The chip vendor is the Endorser
+
+Endorsement:
+
+: The Endorsement only exists virtually inside the chip vendor
+
+Attester:
+
+: The HW and SW made by the chip vendor
+
+Attestation Evidence:
+
+: A protocol message generated by the Attester sent to the cloud service operated by the IoT device vendor
+
+Verifier and Verifier Owner:
+
+: The Verifier and Verifier Owner are split with part run by the chip vendor and part run by the IoT device maker
+
+Relying Party and Relying Party Owner:
+
+: are also the application vendor cloud service; since the Verify and Relying Part are the same the distinction between passport and background check doesn’t apply.
+
+Attestation Result:
+
+: non existent as the Verifier is the same as the Rely Party
+
+## System Example Base on a Third Party Verification Service
+
+Three entities (companies / organizations) participate in this example:
+
+First off, there is a device maker. It does the following:
+* Manufacturers devices using HW and SW of their choice that includes an Attester
+* Puts an attestation key into each device (symmetric or asymmetric)
+* Has the device certified by one or more certification programs
+* Generates a detailed Endorsement for each device that includes the verification key, list of certifications, reference measurement values and implicit claims
+
+The Relying Party operates some cloud service for which attestation needed, perhaps a payment service. It does the following:
+* Receives service requests from devices. The service requests includes Attestation Evidence.
+* Sends the Attestation Evidence to a 3rd party verification service.
+* Receives Attestation Results from 3rd party verification service.
+* Decides whether to grant the service, perhaps varying the payment amount allowed based on details that are in the Attestation Result such the certifications the device has received
+
+The 3rd Party Verification and Attestation Aggregation Service. It does the following:
+* Receives Attestation Evidence from the Relying Party (This uses the Background Check Model)
+* Looks up the Endorsement by an Endorsement identifier, perhaps a URI, and receives the Endorsement from the device maker, most likely over a channel that has both confidentiality, integrity protection and authentication.
+* Performs the Verification of the Attestation Evidence which involves: verifying the signature on the Attestation Evidence and comparison of claims to reference values in the Endorsement.
```

Comment:
```suggestion
* Performs the Verification of the Attestation Evidence which involves: verifying the signature on the Attestation Evidence and comparison of claims to reference values provided along with the Endorsement.
```

Also, if this is intended to be a complete list, it's insecure, since nothing in this sentence does a freshness check for example.